app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Recipe:<%= @recipe.name %></h1>
+<h3>Complexity: <%= @recipe.complexity %></h3>
+<h3>Genre: <%= @recipe.genre %></h3>
+
+<h1>Ingredients:</h1>
+<% @ingredients.each do |ingredient| %>
+  <h3><%= ingredient.name %></h3>
+<% end %>
+
+<h1>Cost: <%= @cost %></h1>
```

Comment:
Here is where I would just use your `@recipe` from the controller and call `@recipe.find_cost`